jekyll/_cci2/sample-config.md

```diff
@@ -31,6 +31,12 @@ This document provides sample [`.circleci/config.yml`]({{ site.baseurl }}/config
 * TOC
 {:toc}
 
+CirlceCI has created an **extension for Visual Studio Code** that reduces context switching for developers between the web app and VS Code through a set of helpful features.
+
+The VS Code extension reduces the time to create, modify, and troubleshoot configuration files through real-time syntax validation, highlighting, and autocomplete suggestions. Authenticating the extension with your CircleCI account will provide you with status alerts as well as the ability to validate your configuration by testing the pipeline locally before you push your code. 
+
+The CircleCI VS Code extension is available to download on the [VS Code marketplace](https://marketplace.visualstudio.com).
```

Comment:
Do we know yet which one it is exactly? I get a few results if I search for CircleCI on https://marketplace.visualstudio.com/vscode.

Reply:
@kbatuigas No, we're waiting on the final link, but nothing else about this will change besides that

Reply:
Apologies, I can finally confirm the link will be https://marketplace.visualstudio.com/items?itemName=circleci.circleci